py-polars/polars/utils/udfs.py

```diff
@@ -472,15 +480,23 @@ def _upgrade_instruction(inst: Instruction) -> Instruction:
         return inst
 
 
-def _is_raw_numpy_function(function: Callable[[Any], Any]) -> bool:
-    """Identify numpy calls that are not wrapped in a lambda/function."""
+def _is_raw_function(function: Callable[[Any], Any]) -> tuple[str, str]:
+    """Identify translatable calls that aren't wrapped inside a lambda/function."""
     try:
-        return (
-            function.__class__.__module__ == "numpy"
-            and function.__name__ in _NUMPY_FUNCTIONS
-        )
+        func_module = function.__class__.__module__
+        func_name = function.__name__
+        if func_module == "numpy" and func_name in _NUMPY_FUNCTIONS:
+            return "np", func_name
+        elif func_module == "builtins" and func_name == "loads":
+            import json  # double-check since it is referenced via 'builtins'
+
+            if function is json.loads:
```

Comment:
legend, mate

Reply:
You know what's funny? I originally didn't have this additional explicit check, but then I asked myself "WWMD" - "What Would Marco Do?". You keep me honest, haha...🤣